transactionoutbox-core/src/main/java/com/gruelbox/transactionoutbox/DefaultMigrationManager.java

```diff
@@ -72,7 +72,18 @@
               8,
               "Update length of invocation column on outbox for MySQL dialects only.",
               "ALTER TABLE TXNO_OUTBOX MODIFY COLUMN invocation MEDIUMTEXT",
-              Map.of(Dialect.POSTGRESQL_9, "", Dialect.H2, "")));
+              Map.of(Dialect.POSTGRESQL_9, "", Dialect.H2, "")),
+          new Migration(
+              9,
+              "Add createTime column to outbox",
+              "ALTER TABLE TXNO_OUTBOX ADD COLUMN createTime TIMESTAMP(6) NULL AFTER invocation",
+              Map.of(
+                  Dialect.POSTGRESQL_9,
+                  "ALTER TABLE TXNO_OUTBOX ADD COLUMN createTime TIMESTAMP(6)")),
+          new Migration(
+              10,
+              "Add groupId column to outbox",
```

Comment:
I wonder whether `partitionId` or `topicId` might be more obvious terms for what this is doing?

`topicId` probably being my favourite.

Reply:
sure, no problem, although these 2 names are fit for kafka/MQs mostly - 99.9% chance that some of that is used with the outbox pattern, but it's theoretically possible to do smth else instead of sending messages (e.g. rest calls)

Reply:
Yeah, I don't mean to imply that `groupId`s _are_ an SQS queue or Kafka topic, only that they are an analog of these, so there's probably no point adding another term to the mix.

Reply:
[Message groups](https://docs.aws.amazon.com/sns/latest/dg/fifo-message-grouping.html) is the terminology AWS SNS/SQS FIFOs use for the boundary within which ordering is ensured, so the term makes sense at least for those users.

Reply:
[Message groups](https://docs.aws.amazon.com/sns/latest/dg/fifo-message-grouping.html) is the term AWS SNS/SQS FIFOs use for the boundary within which ordering is ensured (within a queue or topic), so the “group” term makes sense at least for those users. For Kafka users “partition” would be the equivalent (once again within a topic). So both group or partition seem to be reasonable choices.